source/includes/steps-starting-compass.yaml

```diff
@@ -1,10 +1,17 @@
+title: "Open MongoDB Compass."
+level: 4
+ref: open-compass
+content: |
+  If you need to install |compass-short|, see
+  :ref:`download-install` for instructions.
+---
 title: Enter connection information.
 level: 4
 ref: compass-enter-connection-info
 content: |
 
-    Once you have :doc:`installed </install>` |compass-short| and the
-    program is running, an initial connection dialog appears:
+    When you open |compass-short|, an initial connection dialog
+    appears:
 
     .. figure:: /images/compass/connect-to-host.png
       :figwidth: 626px
```

Comment:
On the TIP admonition: The download center only offers Compass versions as far back as 1.17, so I don't think we need the first part "Starting with version 1.8.0,". Thoughts? I've asked in the #compass-prodmarketing channel as well.

I'd also change:

> and auto-populate the connection dialog from the URI. 

to 

> and auto-populate the connection dialog using the URI. 

Let's also add a double space after the first sentence such that each sentence becomes its own paragraph:

> MongoDB Compass can detect whether you have a MongoDB URI connection string in your system clipboard and auto-populate the connection dialog using the URI. 

> Open MongoDB Compass with a URI connection string in your clipboard and click Yes when prompted to auto-populate the dialog.